src/test/java/org/openrewrite/staticanalysis/LambdaBlockToExpressionTest.java

```diff
@@ -114,4 +117,97 @@ void doTest() {
         );
     }
 
+    //no idea what the problem is, are external libraries not possible in tests?
+    //test output:
+    //java.lang.IllegalStateException: LST contains missing or invalid type information
+    //Identifier->Annotation->MethodDeclaration->Block->ClassDeclaration->CompilationUnit
+    ///*~~(Identifier type is missing or malformed)~~>*/Test
+    //
+    //MethodInvocation->Block->MethodDeclaration->Block->ClassDeclaration->CompilationUnit
+    ///*~~(MethodInvocation type is missing or malformed)~~>*/assertThrows(ArithmeticException.class, () -> {
+    //  BigDecimal.ONE.divide(BigDecimal.ZERO);
+    //})
+    //    at org.openrewrite.java.Assertions.assertValidTypes(Assertions.java:87)
+    //    at org.openrewrite.java.Assertions.validateTypes(Assertions.java:57)
+    //    at org.openrewrite.java.Assertions$$Lambda$438/0x00007fbb18152c28.accept(Unknown Source)
+    //    at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:304)
+    //    at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:132)
+    //    at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:127)
+    //    at org.openrewrite.staticanalysis.LambdaBlockToExpressionTest.simplifyLambdaBlockReturningVoidAsWell(LambdaBlockToExpressionTest.java:121)
+    //    at java.base/java.lang.reflect.Method.invoke(Method.java:568)
+    //    at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
+    //    at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```

Comment:
Yes while it's possible to [specify classpath for dependencies in tests](https://docs.openrewrite.org/authoring-recipes/recipe-testing#specifying-the-classpath-for-dependencies), for this repository it might make most sense to rewrite the test to only use classes available in the JDK; I think that should be manageable here.

If you're curious this shows an example of declaring dependencies on the test classpath:
https://github.com/openrewrite/rewrite-testing-frameworks/blob/a426e409ca3b8e04c6470e86cc09d61346e7c0f4/src/test/java/org/openrewrite/java/testing/cleanup/RemoveTestPrefixTest.java#L32-L33
But again: let's try to avoid that if we can.